frontend/src/index.css

```diff
@@ -497,32 +483,45 @@ Other button classes are defined further down together with other classes for th
   height: 2rem;
 }
 
+.quick-add-wrapper {
+  flex-grow: 3;
+}
+
+.quick-add {
+  align-items: flex-end;
+  display: flex;
+}
+
 .quick-add-input {
-  padding: 0.5rem 0.75rem;
-  margin: 0 0 0 3.75rem;
+  width: 100%;
+  height: 2rem;
+}
+
+.quick-add label {
+  font-size: 0.8rem;
```

Comment:
I think the label here becomes too small, I would remove this line and let it take the default label size

Reply:
fixed